configuration/display_rotation.md

```diff
@@ -0,0 +1,35 @@
+## How to rotate your display
+
+The options to rotate the display of your Raspberry Pi depend on which display driver software it is running, which may also depend on which Raspberry Pi you are using. 
+
+### Fake or Full KMS graphics driver (Default on Pi4)
+
+If you are running the Raspberry Pi desktop then rotation is achieved by using the `Screen Configuration Utility` from the desktop `Preferences` menu. This will bring up a graphical display of the display or displays attached. Right click on the display you wish to rotate and select the required option.
+
+It is also possible to change these settings using the command line `xrandr` option.
+
+```bash
+DISPLAY=:0 xrandr --output HDMI1 --rotate left 
+DISPLAY=:0 xrandr --output HDMI1 --rotate inverted
+DISPLAY=:0 xrandr --output HDMI1 --rotate normal
+```
+
+If you are using the console only (no graphical desktop) then you will need to set the appropriate kernel command line flags. Change the console settings as described on the [this page](./cmdline-txt.md).
+
+### Legacy graphics driver (default on models prior to the Pi4)
+
+There are `config.txt` options for rotating when using the legacy display drivers. 
```

Comment:
Does this rotate **both** the desktop and the console? (probably worth clarifying, given that they're controlled differently when using the KMS driver)